1_fetch.yml

```diff
@@ -167,3 +171,40 @@ targets:
       sb_id = I("5e774324e4b01d509270e29f"), 
       sb_filename = I("pgdl_matched_to_observations.zip"),
       dummy = sb_dl_date)
+  
+  ##-- Access files for GLM3 + GCM projections data release --##
+  
+  # TODO: Switch to downloading the NetCDF file(s) from ScienceBase
+  # to use as input instead. That would be less brittle.
+  
+  # Identifying what lakes / GLM temp output files to use to for 
+  # lake thermal metrics summaries depends on us changing this 
+  # date to refresh the list of files and hashes. Not the most
+  # ideal process, but the function + practicality of this approach
+  # is a bit of a compromise right now so we can complete this work.
+  caldera_access_date:
+    command: c(I('2022-02-16'))
+  
+  # These files from our process-models pipeline are also pushed to ScienceBase
+  #   item `6206d3c2d34ec05caca53071` but since we are running this on Tallgrass
+  #   we have access to Caldera and can skip the need to re-download.
+  pb0_temp_projections_feathers:
+    command: list.files(
+      path = I('../lake-temperature-process-models/3_extract/out'), 
+      pattern = I('GLM_nhdhr_(.*)(GFDL|IPSL|MIROC5|MRI|ACCESS|CNRM).feather'), 
+      full.names = I(TRUE))
+    depends: 
+      - caldera_access_date
+  1_fetch/out/pb0_temp_projections.yml:
```

Comment:
Was going to suggest you combine this target with the `list.files` target, but now I see you are using two base functions and this keeps you from writing a custom function. But something to consider in the future if you run into this since there isn't a clear benefit of having the two be separate (list files is the vector of file names, `sc_indicate` makes it a hash table).

Reply:
Ah, yes. In `targets` I could do both at the same time without a custom function 😄 Custom function might be helpful here since I can name it in a way that describes what's happening.